mono/sgen/sgen-gc.h

```diff
@@ -1111,7 +1111,7 @@ void sgen_env_var_error (const char *env_var, const char *fallback, const char *
 
 /* Utilities */
 
-void sgen_qsort (void *array, size_t count, size_t element_size, int (*compare) (const void*, const void*));
+void sgen_qsort (void *const array, const size_t count, const size_t element_size, int (*compare) (const void*, const void*));
```

Comment:
Top level const is rare on parameters.
Some of these are toplevel, some are not.
I suggest:
`void sgen_qsort(const void* array, size_t, size_t, and then what you have)`
or equivalently
`void sgen_qsort(void const * array, size_t, size_t, and then what you have)`